test/App.spec.js

```diff
@@ -0,0 +1,46 @@
+import { mount } from '@vue/test-utils';
+import app from '../src/App.vue';
+
+describe('App.vue', () => {
+
+  const wrapper = mount(app);
+  it('renders the todo app', () =>{
+    expect(wrapper.contains('div')).toBe(true);
+  });
+
+  //for some unforsaken reason, this one refuses to work.
+  describe('when the toggleEditModeb function is called', () => {
+    const editButton = wrapper.findAll('button').at(2);
+    editButton.trigger('click');
+    it('opens an input field in a listitem component', () => {
+        expect(wrapper.html()).toContain('input');
+    });
+  });
+
+  describe('when the deleteEntryb function is called', () => {
+    wrapper.vm.deleteEntryb(0);
+    it('decreases the amount of items in the list', () => {
+      expect(wrapper.vm.items).toHaveLength(3);
```

Comment:
```suggestion
      expect(wrapper.findAll(ListItem)).toHaveLength(3);
```
`ListItem` is one of your components here.